### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by tuansondinh**
- changed splice to split('.').pop() to get file type
- changed test file

### CODE REVIEW

File: src/PEAPool/PEAPool.ts [hunk 1]
```diff
@@ -76,7 +76,15 @@ abstract class APEAPool implements PEAPool {
...
     public deletePEA(identifier: string, callback: (response: PiMAdResponse) => void): void {
         if(this.initialized) {
-            this.responseHandler.handleCallbackWithResponse(PiMAdResponseTypes.ERROR, '', {}, callback);
+            // find pea by id
+            const localPEA: PEA | undefined  = this.peas.find(pea => identifier === pea.getPiMAdIdentifier());
+            // find index of pea
+            const index = this.peas.indexOf(localPEA as PEA,0);
+            if (index > -1) {
+                this.responseHandler.handleCallbackWithResponse(PiMAdResponseTypes.SUCCESS, 'Success!', {}, callback);
+            }else {
```
Comment (cheidelbach) [inline src/PEAPool/PEAPool.ts:85]: Maybe secure this new lines with a unit test?

**Review by cheidelbach — COMMENTED** [review]
Hey all :) I stumble about the PR and add some comments.

File: src/Converter/Importer/Importer.ts [hunk 2]
```diff
@@ -168,17 +168,17 @@ export class MTPFreeze202001Importer extends AImporter {
      */
     private accessDataSource(instructions: InstructionsConvertFrom, callback: (response: PiMAdResponse) => void): void {
         let gate: Gate = new MockGateFactory().create();
-        switch (instructions.source.slice(-4)) {
-            case '.aml':
+        switch (instructions.source.split('.').pop()) {
```
[the remark this answers is not in the document]
Comment (tuansondinh) [reply src/Converter/Importer/Importer.ts:171]: Julius' idea :) @julorenz

File: src/PEAPool/PEAPool.ts (hunk 1 above)
Comment (tuansondinh) [reply src/PEAPool/PEAPool.ts:85]: Thanks for the hint!

**Pull request closed without merging**